test/stability-tests/src/it/java/software/amazon/awssdk/stability/tests/utils/RetriableTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.stability.tests.utils;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Denotes that a method is a test template for a retriable test.
+ */
+@Target( {ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@TestTemplate
+@ExtendWith(RetriableTestExtension.class)
+public @interface RetriableTest {
+
+    /**
+     * Exception to retry
+     *
+     * @return Exception that handled
+     */
+    Class<? extends Throwable> exception() default Throwable.class;
```

Comment:
Can we do retryException, or something the clarify that this is the retry condition?

Reply:
yeah, will do